.gitlab-ci/packet.yml

```diff
@@ -24,7 +24,7 @@ packet_ubuntu20-calico-aio:
 
 packet_centos7-flannel-addons:
   extends: .packet
-  stage: deploy-part2
```

Comment:
may be need revert back ???

Reply:
Yes will do,thats why wip flag is still there